week3/Menu-Recommender/src/components/RecommendOption.jsx

```diff
@@ -0,0 +1,157 @@
+import styled from "styled-components";
+
+const RecommendOption = ({ setOptionType, optionType, setStep }) => {
+  const handleClickStartButton = () => {
+    setStep(0);
+  };
+
+  return (
+    <RecommendOptionWrapper>
+      <RecommendOptionHeader>
+        <h1>원하는 추천 방식을 골라줘!</h1>
+      </RecommendOptionHeader>
+      <RecommendOptionButtonWrapper>
+        {optionType === 1 ? (
+          <SelectResultBox>
+            <h2>
+              취향대로
+              <br />
+              추천
+            </h2>
+          </SelectResultBox>
+        ) : optionType === 2 ? (
+          <SelectResultBox>
+            <h2>
+              랜덤
+              <br />
+              추천
+            </h2>
+          </SelectResultBox>
+        ) : (
+          <>
+            <RecommendOptionButton onClick={() => setOptionType(1)}>
+              <h2>
+                취향대로
+                <br />
+                추천
+              </h2>
+            </RecommendOptionButton>
+            <RecommendOptionButton onClick={() => setOptionType(2)}>
+              <h2>
+                랜덤
+                <br />
+                추천
+              </h2>
+            </RecommendOptionButton>
+          </>
+        )}
+      </RecommendOptionButtonWrapper>
+      {optionType !== -1 ? (
+        <StartButton onClick={handleClickStartButton}>시작하기</StartButton>
+      ) : (
+        <></>
+      )}
```

Comment:
```suggestion
      {optionType !== -1 &&
        <StartButton onClick={handleClickStartButton}>시작하기</StartButton>
      }
```

요기 삼항연산자에서 optionType이 -1일때에 빈태그를 반환하는데 이런 경우엔 && 연산자를 활용해서 나타내줘도 될 거 같어!!